patches/net/minecraft/world/entity/item/ItemEntity.java.patch

```diff
@@ -135,6 +135,21 @@
          if (!this.level().isClientSide && entity instanceof ItemEntity) {
              ((ItemEntity)entity).mergeWithNeighbours();
          }
+@@ -397,6 +_,14 @@
+         this.target = p_266724_;
+     }
+ 
++    /**
++     * Neo: Add getter for ItemEntity's {@link ItemEntity#target target}.
++     * @return The {@link ItemEntity#target target} that can pick up this item entity (if {@code null}, anyone can pick it up)
++     */
++    public @org.jetbrains.annotations.Nullable UUID getTarget() {
```

Comment:
Typically we place all annotations above the method declaration instead of inlining them, i.e.:
```java
@org.jetbrains.annotations.Nullable
public UUID getTarget() {
```

Reply:
Also, this class already imports the javax nullable, so we should use that instead of the jetbrains one for consistency

Makes the patch look a bit cleaner too